src/Command/ModelCommand.php

```diff
@@ -689,11 +689,12 @@ public function getValidation(Table $model, array $associations, Arguments $args
             }
         }
         foreach ($fields as $fieldName) {
-            if (in_array($fieldName, $foreignKeys, true)) {
+            // Skip primary key and foreign key validations
+            if (in_array($fieldName, $primaryKey, true) || in_array($fieldName, $foreignKeys, true)) {
```

Comment:
Why are you skipping foreign key validation?

Reply:
This was present before as well, don't know why 🤷🏻

Reply:
It is there from the start
https://github.com/cakephp/bake/commit/12dea6d0e168031c6f939ce6adffa29943f5783e#diff-b2a7f0a392cef7628e2952ea340e286e957a809321cbe9cb98047925b2e83e8bR667
Maybe the primary key part was meant all along?

Reply:
@LordSimal Sorry, missed the fact that it already existed.

Though I wonder why, since fKs can be part of mass assignment and should be validated.

Reply:
@dereuromark It's the same in the old task too, so if it's a mistake it's existed for a long time.
https://github.com/cakephp/bake/blob/33e8ee8419ba36c13fa4074c208c93352b5530cf/src/Shell/Task/ModelTask.php#L679

Reply:
If you desire I can change that to only skip on PK but not on FK

Reply:
Its funny because there is an extra Test just to check that FK are not included int the generated validation 😂 
https://github.com/cakephp/bake/blob/2.x/tests/TestCase/Command/ModelCommandTest.php#L1088